bin/tachyon-workers.sh

```diff
@@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
 fi
 
 bin=`cd "$( dirname "$0" )"; pwd`
-DEFAULT_LIBEXEC_DIR="$bin"/../libexec
```

Comment:
as @caadecarvalho mentioned, when you run `${TACHYON_HOME}/bin/tachyon-mount.sh SudoMount workers`, the `$bin` environment variable is never set and so the script does not do the right thing